docs/discover/kuery.asciidoc

```diff
@@ -3,22 +3,32 @@
 
 experimental[This functionality is experimental and may be changed or removed completely in a future release.]
 
+[NOTE]
+============
+Breaking changes were made to Kuery's experimental syntax in 6.3. Read on for details of the new syntax.
+============
+
 Kuery is a new query language built specifically for Kibana. It aims to simplify the search experience in Kibana
 and enable the creation of helpful features like auto-complete, seamless migration of saved searches, additional
 query types, and more. Kuery is a basic experience today but we're hard at work building these additional features on
 top of the foundation Kuery provides.
 
-Kueries are built with functions. Many functions take a field name as their first argument. Extremely common functions have shorthand notations.
+If you're familiar with Kibana's old lucene query syntax, you should feel right at home with Kuery. Both languages 
```

Comment:
Are we calling it Kuery or KQL?  The PR description makes it sound like we are renaming it KQL, but here it's still referred to as Kuery.

Reply:
Yeah sorry, it's a bit confusing. We've been calling the simple language "KQL" in the code and amongst ourselves to distinguish it from the old complex grammar. It'll still be called "kuery" in the UI when this PR is merged, but the plan is ultimately to remove the language dropdown and just add a checkbox to opt-in to "query language enhancements", which will turn on the simple language hopefully without it seeming like a big jump to a whole different language. At that point we can get rid of the kuery name, it just didn't make sense to us to introduce yet another new language to the language switcher only to take it away soon after.